src/k8s/pkg/k8sd/features/cilium/status.go

```diff
@@ -0,0 +1,34 @@
+package cilium
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/canonical/k8s/pkg/snap"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CheckNetwork(ctx context.Context, snap snap.Snap) (bool, error) {
+	client, err := snap.KubernetesClient("")
```

Comment:
nit: We only operate in the `kube-system` namespace in this function, hence the client namespace could be limited to `kube-system`